src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -252,68 +279,71 @@ public RealJenkinsRule withHost(String host) {
                     File plugins = new File(home, "plugins");
                     plugins.mkdir();
                     FileUtils.copyURLToFile(RealJenkinsRule.class.getResource("RealJenkinsRuleInit.jpi"), new File(plugins, "RealJenkinsRuleInit.jpi"));
-                    // Adapted from UnitTestSupportingPluginManager & JenkinsRule.recipeLoadCurrentPlugin:
-                    Set<String> snapshotPlugins = new TreeSet<>();
-                    Enumeration<URL> indexJellies = RealJenkinsRule.class.getClassLoader().getResources("index.jelly");
-                    while (indexJellies.hasMoreElements()) {
-                        String indexJelly = indexJellies.nextElement().toString();
-                        Matcher m = SNAPSHOT_INDEX_JELLY.matcher(indexJelly);
-                        if (m.matches()) {
-                            Path snapshotManifest;
-                            snapshotManifest = Paths.get(URI.create(m.group(1) + "/test-classes/the.jpl"));
-                            if (!Files.exists(snapshotManifest)) {
-                                snapshotManifest = Paths.get(URI.create(m.group(1) + "/test-classes/the.hpl"));
-                            }
-                            if (Files.exists(snapshotManifest)) {
-                                String shortName;
-                                try (InputStream is = Files.newInputStream(snapshotManifest)) {
-                                    shortName = new Manifest(is).getMainAttributes().getValue("Short-Name");
-                                }
-                                if (shortName == null) {
-                                    throw new IOException("malformed " + snapshotManifest);
+
+                    if (skipAllPlugins) {
```

Comment:
because my previous comment, shouldn't be the opposite?
```suggestion
                    if (!skipAllPlugins) {
```